Dockerfile

```diff
@@ -16,4 +16,6 @@ RUN pip install -e .
 # Create and switch to a new non-root user
 RUN useradd worker
 RUN chown worker:worker -R /home/worker
+
+RUN apt-get update && apt-get install -y curl
```

Comment:
Why are you installing curl in the container. Doesn't look like you are using in docker-compose anyway..